provisionersdk/proto/provisioner.pb.go

```diff
@@ -2168,6 +2168,7 @@ type Module struct {
 	Source  string `protobuf:"bytes,1,opt,name=source,proto3" json:"source,omitempty"`
 	Version string `protobuf:"bytes,2,opt,name=version,proto3" json:"version,omitempty"`
 	Key     string `protobuf:"bytes,3,opt,name=key,proto3" json:"key,omitempty"`
+	Dir     string `protobuf:"bytes,4,opt,name=dir,proto3" json:"dir,omitempty"`
```

Comment:
@spikecurtis we don't have an `apiversion` defined for this currently; do you see this being an issue? I see that we've made previous additions to this file and I can't see corresponding increments in any corresponding version?